channels/chan_voter.c

```diff
@@ -3903,8 +3905,8 @@ static void *voter_reader(void *data)
 			continue;
 		}
 		vph = (VOTER_PACKET_HEADER *) buf;
-		ast_debug(6, "Got rx packet, len %d payload %d challenge %s digest %08x\n", (int) recvlen, ntohs(vph->payload_type),
-			vph->challenge, ntohl(vph->digest));
+		ast_debug(7, "Got RX packet, len %d payload %d challenge %s digest %08x from client %s\n", (int) recvlen,
```

Comment:
I guess I should have looked at this a bit closer:
We don't know the client at this point in the code so we can't get valid client info yet.